src/containers/Sidebar/index.js

```diff
@@ -19,6 +20,13 @@ const Sidebar = (props) => {
 
 	let sidebarItems = props.navigation.getItems().items;
 
+	const BrandingService = props.app.Services.BrandingService;
```

Comment:
@petrKavulok As per my comment above. Btw. you are using this same approach on 5 places (across 5 files), maybe its worth to consider creating a reusable component that this code does not repeat itself.